django_project/certification/templates/course/detail.html

```diff
@@ -75,6 +75,7 @@ <h1>{{ course.course_type.name }} ({{ course.start_date }} to {{ course.end_date
             <div class="details col-lg-10" style="width: 60%">
                 <span class="grey-italic col-lg-3">Course name</span><span class="col-lg-9">{{ course.course_type.name }} ({{ course.start_date }} to {{ course.end_date }})<br/></span>
                 <span class="grey-italic col-lg-3">Course type</span><span class="col-lg-9">{{ course.course_type }}<br/></span>
+                <span class="grey-italic col-lg-3">Course Language</span><span class="col-lg-9">{{ course.language }}<br/></span>
```

Comment:
Course language. The other use sentence case. Just make sure to have the same case.